browser/resources/settings/email_aliases_page/tests/email_aliases_modal.test.tsx

```diff
@@ -188,8 +188,9 @@ describe('EmailAliasModal', () => {
   })
 
   it('shows loading state while generating alias', async () => {
+    const aliasEmail = 'new@brave.com'
     mockEmailAliasesService.generateAlias = jest.fn().mockImplementation(
-      () => Promise.resolve('new@brave.com'))
+      () => Promise.resolve({ result: { errorMessage: null, aliasEmail } }))
```

Comment:
@fallaciousreasoning is this the correct mapping from mojo interface for optional? It will still have the property and just be null vs not having the property at all?

Reply:
Good catch. Somehow the typescript linter wasn't catching this until I explicitly added the type there. Actually it maps to

```
/**
 * @typedef { {
 *   aliasEmail: (!string|undefined),
 *   errorMessage: (!string|undefined),
 * } }
 */
export const GenerateAliasResult = {};
```


So I needed to use undefined rather than null. I have pushed a fix

Reply:
Nice catch :smile:

Reply:
I have a vague idea that the property has to be present, it just also has to be undefined.